vcpkg.json

```diff
@@ -107,7 +107,7 @@
             "dependencies": [
                 {
                     "name": "onnx",
-                    "version>=": "1.13.1"
+                    "version>=": "1.14.1"
```

Comment:
in current master we have:
https://github.com/openvinotoolkit/openvino/blob/8eb165021c898d6078e45580b991c50bd6e7a998/vcpkg.json#L160-L164
which needs to be removed because we are updating version in current PR.

BTW, latest version in vcpkg is 1.14.0
https://github.com/microsoft/vcpkg/blob/master/versions/o-/onnx.json#L4-L7

Maybe we need to relax `find_package(ONNX)` and remove exact? Because different package managers add new versions not simultaneously and we can always have a gap.
Note, that vcpkg and Conan build are just for CI validation and we don't redistribute artifacts. So, their build with different ONNX versions is OK, IMO.

Reply:
Please, take a look